pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -133,7 +133,7 @@ func newAlertmanager(ctx context.Context, orgID int64, cfg *setting.Cfg, store A
 		return nil, err
 	}
 
-	am := &Alertmanager{
```

Comment:
From this point on (almost) all changes to this file are `Alertmanager -> alertmanager`